yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/SecureYotiClient.java

```diff
@@ -162,54 +171,38 @@ private Key parseKey(byte[] keyVal) {
         return new SecretKeySpec(keyVal, SYMMETRIC_CIPHER);
     }
 
-    private Map<String, Object> parseProfile(byte[] profileData) throws ProfileException {
```

Comment:
Since we're here...
```
private List<Attribute> parseProfile(byte[] profileData) throws ProfileException {
        try {
            AttributeList message = AttributeList.parseFrom(profileData);
            List<Attribute> attributeList = parseAttributes(message);
            LOG.debug("{} attribute(s) parsed", attributeList.size());
            return attributeList;
        } catch (InvalidProtocolBufferException e) {
            throw new ProfileException("Cannot parse profile data", e);
        }
    }
```